examples/energy-management-app/esp32/main/main.cpp

```diff
@@ -117,10 +125,18 @@ namespace Clusters {
 namespace DeviceEnergyManagement {
 
 // Keep track of the parsed featureMap option
+#if 1 // 0x7a
```

Comment:
Make this a config flag

Reply:
Changed with a number of #defines in the Kconfig.projbuild file